<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_gestaomarcas</class>
 <widget class="QDialog" name="frm_gestaomarcas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>403</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Nova Marca</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>10</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Dados Marca:</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>144</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Marca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_marca"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>396</y>
       <width>186</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_nv_novo">
        <property name="text">
         <string>Novo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_nv_salvar">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget_2</zorder>
    <zorder>groupBox</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gest√£o Marcas</string>
    </attribute>
    <widget class="QTableWidget" name="tw_listamarcas">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>67</y>
       <width>381</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>269</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Dados Marca:</string>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>144</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Marca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ge_marca"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>398</y>
       <width>168</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_ge_salvar">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_excluir">
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>34</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="txt_ge_filtrar"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_filtrar">
        <property name="text">
         <string>filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Filtro:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_ge_filtrar"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
